e2e_test.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+log() {
+  echo -e "\n>>>>> $1\n"
+}
+
+fail() {
+  echo "$1"
+  exit 1
+}
+
+# Wrapping every command in this, in order to avoid a cascasing pile of errors
+try_command() {
+  eval "$1" || fail "This command failed: '$1'"
+}
+
+####################################################################################################
+
+# This list should be extended as more demos are written!
+directories=(
+  bank/bank-backend
+  # bank/bank-frontend
+
+  e-commerce/payment-backend
+  e-commerce/shop-backend
+  e-commerce/shop-frontend
```

Comment:
Not a DBOS app

Reply:
Would still be good to test the linter here, regardless, I think